perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -370,14 +401,14 @@ def WriteObjects(storage_schema, bucket, object_prefix, count,
         latency_results.append(latency)
       if bandwidth_results is not None and latency > 0.0:
         bandwidth_results.append(size / latency)
+      handle.seek(0)
```

Comment:
wow.... seek is buried here. I'd strongly consider move the seek() to immediately before the place where the handle is going to be read,  that is, inside WriteObjectFromBuffer()